src/Model/Command/UserRecommendation.php

```diff
@@ -279,17 +286,26 @@ protected function getCommandParameters(): array
     {
         $parameters = [
             'user_id' => $this->userId,
-            'count' => $this->count,
             'scenario' => $this->scenario,
-            'rotation_rate' => $this->rotationRate,
-            'rotation_time' => $this->rotationTime,
             'hard_rotation' => $this->hardRotation,
             'min_relevance' => $this->minimalRelevance->jsonSerialize(),
             'filter' => $this->assembleFiltersString(),
             'filter_type' => $this->filterType,
             'properties' => $this->responseProperties,
         ];
 
+        if ($this->count !== null) {
```

Comment:
What about other params? hard_rotation, min_relevance, filter, filter_type, properties

They will always be added to the request, even if not set by client. Is this intentional?

Reply:
No, it's an oversight. Fixed